CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 Lists all changes with user impact.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+## [0.19.37]
```

Comment:
I think upgrading major version of spring-boot should at least change minor version of envoy-control